### PR TITLE
combined css to bb_style.css 

### LINKED ISSUE #53: consolidating css continued.
consolidating merch, social media, contact, and FAQ into one main css for the entire site.

### PR DESCRIPTION
**Pull request by margeschrec**
closes #53 (accidently named branch 57, meant to name it 5)

Incorporated css from the separate css stylesheets (press, social media, contact, and FAQ) into the bb_style.css. 
I added some class specificity to card classes so the merchandise page would be able to be manipulated a part from the card links on the press and social media pages.

### CODE REVIEW

**Review by vricci518 — APPROVED** [review]

File: Merch.php [hunk 1]
```diff
@@ -6,32 +6,32 @@
 	<title>BAR-BILL TAVERN - Bar-Bill Tavern - Restaurant in NY</title>
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Oswald">
-	<link rel="stylesheet" href="css/barbill.css">
-	<link rel="stylesheet" href="css/merch.css">
+    <link rel="stylesheet" href="css/bb_style.css">
```
Comment (billythakidd04) [inline Merch.php]: need to fix the indentation

File: contact.php [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
+    <link rel="stylesheet" href="css/bb_style.css">
+	<link rel="stylesheet" href="css/bb_common.css">
 </head>
 
 <body>
 <?php include 'header.php'; ?>
 	<div class="container">
 		</br>
-		<div id="CUH">
+		<div class="cuh">
```
Comment (billythakidd04) [inline contact.php]: what is cuh!?

File: css/bb_style.css [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /* Home */
 
-.reservation {
+.reservation, {
```
Comment (billythakidd04) [inline css/bb_style.css]: why the comma?

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
2 things:

1. css syntax should be:

```CSS
tag {
    key: value;
}
/* blank line here*/
nextTag {
    key: value;
}
```

2. Some of the naming conventions are a bit verbose while others are complete obfuscation. Keep it simple but self documenting. And the files don't need the bb_ prefix. If a potential employer sees that, it's an immediate red flag. It tells us this person is concentrating on the wrong things.

File: contact.php (hunk 2 above)
Comment (margeschrec) [reply contact.php]: ah, contact us header- I see your point.

**Review by billythakidd04 — APPROVED** [review]

**Review by vricci518 — APPROVED** [review]

**Pull request merged**